core/src/main/java/org/jboss/set/mjolnir/archive/ldap/LdapScanningBean.java

```diff
@@ -203,4 +195,38 @@ private void createUniqueUserRemoval(Set<String> existingUserNamesToProcess, Str
             em.persist(removal);
         }
     }
+
+    private List<RegisteredUser> getRegisteredUser() {
```

Comment:
This method is quite general and it would be handy to make it available project wide.

You could move it to `RegisteredUserRepositoryBean` and make it public. Then just inject the repository bean in here.

Maybe also rename it to `getAllUsers()` - plural.